agent/server/server.go

```diff
@@ -86,7 +87,7 @@ func Start() {
 			ClientAuth:   tls.RequireAnyClientCert,
 		}
 		business.Init()
-		global.LOG.Info("listen at https://0.0.0.0:9999")
+		global.LOG.Infof("listen at https://0.0.0.0:%s", global.CONF.System.Port)
 		if err := server.ListenAndServeTLS("", ""); err != nil {
 			panic(err)
 		}
```

Comment:
This is a snippet of C++ code related to web development, specifically using the HTTP server protocol over TCP on Unix sockets. The main differences between the original version from GitHub and a potentially updated version are:

The `github.com` imports have been replaced with more specific ones (specifically, replacing 'crypto/tls' and '.net') which could improve readability when importing third-party packages.

No other discrepancies appear; this appears to be an example where certain dependencies or paths within the original project were changed slightly compared to its source but without altering functionality. In particular, I don't see any major changes in logic or operations that would suggest problems like security vulnerabilities or runtime errors unless there's significant difference in behavior. However, minor adjustments might help prevent bugs especially after long time since it was last released.

As always, make sure this has actually been tested for real-world use before making substantial updates!